src/tool/xmeta/main.cpp

```diff
@@ -6,25 +6,50 @@
 
 #include "xmeta_idl_reader.h"
 #include "ast_to_st_listener.h"
+#include "xlang_model_walker.h"
+#include "xmeta_emit.h"
 
 #if defined(_WIN32)
```

Comment:
There isn't any alternative on non-Windows platforms for this. Instead, I think it makes more sense to simply restrict the entire xmeta tool executable to only WIN32 in CMakeFiles.txt.